Sources/Private/NodeRenderSystem/NodeProperties/ValueProviders/GroupInterpolator.swift

```diff
@@ -9,7 +9,7 @@ import CoreGraphics
 import Foundation
 
 /// A value provider that produces an array of values from an array of Keyframe Interpolators
-final class GroupInterpolator<ValueType>: AnyValueProvider where ValueType: Interpolatable {
+final class GroupInterpolator<ValueType>: ValueProvider where ValueType: Interpolatable {
```

Comment:
I refactored the existing `ValueProvider`s to support checking whether or not the provider contains a single value or a closure that is intended to be called every frame (which can't really but supported by the new rendering engine)